Application/data_payload.hpp

```diff
@@ -144,6 +148,51 @@ struct DataPayload {
 						air_temp_,
 						coolant_temp_);
     }
+    void RawArray(float* raw_buffer) {
+		float raw_array[] = {
+				timestamp_,
+
+				linpot_displacement_mm_[0],
+				linpot_displacement_mm_[1],
+				linpot_displacement_mm_[2],
+				linpot_displacement_mm_[3],
+
+				acceleration_[0],
+				acceleration_[1],
+				acceleration_[2],
+
+				angular_velocity_[0],
+				angular_velocity_[1],
+				angular_velocity_[2],
+
+				rpm_,
+				tps_,
+				fuel_open_time_,
+				ignition_angle_,
+
+				barometer_,
+				map_,
+				lambda_,
+
+				analog_inputs_[0],
+				analog_inputs_[1],
+				analog_inputs_[2],
+				analog_inputs_[3],
+				analog_inputs_[4],
+				analog_inputs_[5],
+				analog_inputs_[6],
+				analog_inputs_[7],
+
+				battery_voltage_,
+				air_temp_,
+				coolant_temp_
+		};
+
+		for (size_t i = 0; i < 20; ++i) {
```

Comment:
May be a good idea to compute the length of the array

Reply:
I had it calculated at compile time originally, but changed it to a static value because I was running into issues somewhere in the program. I agree it's probably a good idea